### PR TITLE
Fix FileSystemAccess tests

### PR DESCRIPTION
**Pull request by s-tarasov**

### CODE REVIEW

**Comment by ferventcoder** [comment]
Breaking it up?

File: product/roundhouse.tests/infrastructure/filesystem/WindowsFileSystemAccessSpecs.cs [hunk 1]
```diff
@@ -49,13 +50,20 @@ public class when_reading_files_with_different_formats : concern_for_file_system
...
+                utf8_file.should_be_equal_to("INSERT INTO [dbo].[timmy]([value]) VALUES('Gã')");
             }
 
             [Observation]
             public void ansi_encoded_file_should_read_correctly()
+            {   
+                ansi_file.should_be_equal_to(to_default_codepage("INSERT INTO [dbo].[timmy]([value]) VALUES('Gã')"));
+            }
+
+            private string to_default_codepage(string englishCodepageString)
```
Comment (ferventcoder) [inline product/roundhouse.tests/infrastructure/filesystem/WindowsFileSystemAccessSpecs.cs]: Almost all good, except for the variable name. Look at the others. Use the same format. Thanks!

**Comment by s-tarasov** [comment]
Oh break it up, yeah break it up,
Oh oh break it up, yeah just break it up))

**Comment by BiggerNoise** [comment]
Because of merge conflicts, I went ahead and recreated this commit. Thank you for the contribution.

**Pull request closed without merging**